pallets/runtime/testnet/src/runtime.rs

```diff
@@ -462,7 +481,7 @@ impl pallet_contracts::Trait for Runtime {
     type Randomness = RandomnessCollectiveFlip;
     type Currency = Balances;
     type Event = Event;
-    type Call = Call;
+    //type Call = Call;
```

Comment:
Do we need this comment? If there is no `Call` then it could be removed.

Reply:
I'll clean it up in a bit; thanks.

Reply:
Fixed.